be/src/exec/workgroup/work_group.h

```diff
@@ -116,6 +122,19 @@ class WorkGroup {
         auto now = duration_cast<milliseconds>(steady_clock::now().time_since_epoch()).count();
         return now > _vacuum_ttl;
     }
+
+    int64_t total_cpu_cost() const { return _total_cpu_cost.load(); }
+    void incr_total_cpu_cost(int64_t cpu_cost) { _total_cpu_cost.fetch_add(cpu_cost); }
+
+    int64_t total_io_cost() const { return _total_io_cost; }
+    void incr_total_io_cost(int64_t io_cost) { _total_io_cost += io_cost; }
+
+    bool is_big_query(const QueryContext& query_context);
+    bool is_check_big_query() { return _big_query_limit == 1 ? false : true; }
```

Comment:
Maybe just return `_big_query_limit != 1`?

Reply:
_big_query_limit was a double. Is it possible to cause accuracy problems when comparing with 1